highfidelity/hifi
# Smooth other avatars positions [Experiment]

This PR interpolate the position of other avatars when the distance  between frames is more than a certain value. This could be useful when:
- Teleporting
- Slow avatar updates.

## TEST PLAN
- Use this app to configure the parameters: https://hifi-content.s3.amazonaws.com/luis/test_scripts/transitApp/transitApp.js
- Go to a domain with other avatars.
- When they teleport you should see them moving in a straight line to their destination.
- When you teleport you should move instantaneously to your destination (default behavior)
